sklearn/inspection/tests/test_plot_partial_dependence.py

```diff
@@ -132,9 +141,15 @@ def test_plot_partial_dependence_str_features(pyplot, clf_boston, boston,
     assert disp.axes_.shape == (2, 1)
     assert disp.lines_.shape == (2, 1)
     assert disp.contours_.shape == (2, 1)
+    assert disp.deciles_vlines_.shape == (2, 1)
+    assert disp.deciles_hlines_.shape == (2, 1)
 
     assert disp.lines_[0, 0] is None
     assert disp.contours_[1, 0] is None
+    assert disp.deciles_vlines_[0, 0] is not None
```

Comment:
Nit: Reorder to keep `[0, 0]` and `[1, 0]` together:

```py
assert disp.lines_[0, 0] is None
assert disp.deciles_vlines_[0, 0] is not None
assert disp.deciles_hlines_[0, 0] is not None

# etc
```

Reply:
done